path_login.go

```diff
@@ -411,8 +417,8 @@ type k8sObjectRef struct {
 
 // lookup calls the TokenReview API in kubernetes to verify the token and secret
 // still exist.
-func (s *serviceAccount) lookup(ctx context.Context, jwtStr string, tr tokenReviewer) error {
-	r, err := tr.Review(ctx, jwtStr, s.Audience)
+func (s *serviceAccount) lookup(ctx context.Context, jwtStr string, tr tokenReviewer, client *http.Client) error {
```

Comment:
It'd be nice to also have the `*http.Client` go right after `context.Context` in here, similar to how we have it for the `tokenReviewAPI.Review` method.

Reply:
Updated in https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/142/commits/e1765a3788f49f37fbf426d7ad8025613496d7a5